src/hooks/useSwapCurrencyList.ts

```diff
@@ -94,6 +106,20 @@ const useSwapCurrencyList = (
   const [lowLiquidityAssets, setLowLiquidityAssets] = useState<RT[]>([]);
   const [verifiedAssets, setVerifiedAssets] = useState<RT[]>([]);
 
+  const { inputCurrency } = useSwapCurrencies();
+  const crosschainSwapsEnabled = useExperimentalFlag(CROSSCHAIN_SWAPS);
+  const inputChainId = useMemo(
+    () =>
+      inputCurrency?.network &&
+      ethereumUtils.getChainIdFromNetwork(inputCurrency?.network as Network),
+    [inputCurrency?.network]
+  );
+  const isCrosschainSearch = useMemo(() => {
+    if (inputChainId && inputChainId !== chainId && crosschainSwapsEnabled) {
```

Comment:
this is just a naming comment. it took me a second to know where `chainId` was coming from and figured out is coming from currency select list network, not input nor output which makes sense, what do you think of changing the name internally to something like `searchChainId` or something? to make it clear that this value is for searching and not coming from tokens as i thought at the beginning

Reply:
for ex. when i saw `fromChainId` i immediately assumed it was coming from the input